drivers/Makefile.dep

```diff
@@ -608,6 +608,23 @@ ifneq (,$(filter w5100,$(USEMODULE)))
   USEMODULE += luid
 endif
 
+ifneq (,$(filter ws281x_%,$(USEMODULE)))
+  USEMODULE += ws281x
+endif
```

Comment:
This seems pretty redundant.

Reply:
If multiple backends are available, the user might was to explicitly add a specific one. E.g. when the SPI backend gets added, this could be used as well on ATmega. Without this a user would have to add both `ws281x_spi` and `ws281x`.

Reply:
Yes I just noticed I contrasted myself by proposing both an SDL and an ASCII backend for native…